mathematical/linear_regression.py

```diff
@@ -49,12 +49,20 @@
 #  |  DOI: `10.1021/acs.jproteome.8b00717 <http://dx.doi.org/10.1021/acs.jproteome.8b00717>`_
 #
 
+# stdlib
+from typing import Tuple
+
 # 3rd party
-import numpy
-from domdf_python_tools.doctools import is_documented_by
+import numpy  # type: ignore
+from domdf_python_tools.doctools import is_documented_by  # type: ignore
 
 
-def linear_regression_vertical(x, y=None, a=None, b=None):
+def linear_regression_vertical(
+		x: numpy.ndarray,
+		y: numpy.ndarray = None,
```

Comment:
Can you make `y` `Optional[numpy.ndarray]`, and `a` and `b` `Optional[float]` please?